assets/styles/generator.scss

```diff
@@ -59,7 +59,7 @@ textarea:placeholder-shown + label.fancy-label {
 /* Inputs */
 .dynamic-input-controls {
     display: flex;
-    justify-content: center;
```

Comment:
This breaks the dropup:
![Screen Shot 2022-07-15 at 12 33 41](https://user-images.githubusercontent.com/4049135/179206720-e0e25d62-a401-4d1d-88d9-a18e125788e9.png)